src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/BarometerReading.cs

```diff
@@ -2,12 +2,12 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.Devices.Sensors
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+#if false || false || false || false || false
```

Comment:
💅 minor nitpick, that doesn't require a change. Similar story here. This class could have been completely stripped out.